### ISSUE TITLE
Programm stucks

### ISSUE THREAD

**Issue by kyrtka** [gephi/gephi#1473]
it Stucks every 5 seconds, impossible to work! doesn't show the nodes't titles sometimes, doesn't allow to change the edges size sometimes, and sometimes it just stop working and i can't turn it off at all, and its very often! I've deleted and uploaded it again for several times but still it works like this. I have windows 8.

**Comment by eduramiba** [comment]
Sorry, we have some stability issues, specially with some datasets with parallel edges for example.
Does this happen with all files?

**Comment by kyrtka** [comment]
No, before it didn't happen with ladge files, but as soon as i started to load small files, it all has been stucked. -- C уважением, Береснева Дарья+7-985-288-82-81   26.04.2016, 11:31, "Eduardo Ramos" <notifications@github.com>:Sorry, we have some stability issues, specially with some datasets with parallel edges for example. Does this happen with all files?—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub

**Comment by eduramiba** [comment]
Has it improved for you with latest 0.9.2 snapshot? https://github.com/gephi/gephi#nightly-builds
